cg/models/demultiplex/flow_cell.py

```diff
@@ -87,26 +90,6 @@ def get_sample_sheet_path_hk(self) -> Path:
             raise FlowCellError("Attribute _sample_sheet_path_hk has not been assigned yet")
         return self._sample_sheet_path_hk
 
-    @property
-    def run_parameters_path(self) -> Path:
-        """Return path to run parameters file."""
-        return Path(self.path, DemultiplexingDirsAndFiles.RUN_PARAMETERS)
-
-    @property
-    def run_parameters(self) -> RunParameters:
```

Comment:
Does these more belong to how the flow cell was processed than downstream demux (and potential other processes)?